docs/dev/setup/local-database-tests.rst

```diff
@@ -0,0 +1,37 @@
+.. _local_database_tests:
+
+Running database tests locally
+---------------------------------
+
+Prerequisites
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Database tests are standard server integration tests executed against real database (MySQL or Postgres) started as Docker containers. Gradle test task will automatically pull necessary images and start Docker containers.
+
+Before running those tests make sure that the following steps were executed:
+
+1. Install `Docker Desktop <https://docs.docker.com/desktop/#docker-for-mac>`__ or `Docker Engine and Docker CLI <https://docs.docker.com/engine/install/>`__.
+2. Enable `remote access for Docker daemon <https://docs.docker.com/config/daemon/remote-access/>`__.
+
+Executing tests from Intellij
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Database tests can be executed by running of the run configurations provided in the repository:
```

Comment:
Please consider not using passive voice and therefore rephrasing the "Database tests..." sentence.
In the same sentence "of" is not needed. You can remove it.